ballista/rust/scheduler/src/scheduler_server/event_loop.rs

```diff
@@ -70,12 +70,28 @@ impl<T: 'static + AsLogicalPlan, U: 'static + AsExecutionPlan>
             return Ok(Some(SchedulerServerEvent::JobSubmitted(job_id)));
         }
 
+        let mut executors_delta_data: Vec<ExecutorDeltaData> = available_executors
```

Comment:
`executors_delta_data` is confused for me, because this just clone/copy some attributes from `available_executors`

Reply:
`ResourceChange` or `SlotsChange` probably?

Reply:
fixed